Exercise_2/interactive_command_list

```diff
@@ -2,9 +2,9 @@
 srun --partition=express --nodes=1 --cpus-per-task=2 --pty --time=00:30:00 /bin/bash
 
```

Comment:
You might want to include: 

#Initial Spack setup
git clone -c feature.manyFiles=true https://github.com/spack/spack.git
#